packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/useDatabases.ts

```diff
@@ -66,8 +98,23 @@ export function useDatabases() {
 
   return {
     connect,
-    dbs,
-    syncStatus,
+    fetchAttempt,
+    dbs: fetchAttempt.data?.agentsList,
```

Comment:
Sorry that I didn't catch that while I was reviewing `useServers` but I don't think extracting stuff out of `fetchAttempt` is worthwhile long term.

Ideally the hook would expose `fetchAttempt` and then any UI code that needs to work on that data would look at `fetchAttempt`.

At the moment, we return four different values that are based on `fetchAttempt`:

* `fetchAttempt`
* `dbs` - the view can read `fetchAttempt.data?.agentsList || []` instead.
   * Ideally, the `Table` component would work with `Attempt<T>` out of the box. But I understand if we decided not to do it right now.
* `disabledRows` - this can be "calculated" within the component that calls `useDatabases`.
* `emptyTableText` - same as above.

I think the idea is that the hook returns the minimal amount of data required to display the UI and then the component that calls `useDatabases` works with that data to render the UI.

Theoretically, a custom hook like that could be reused in another page but right now it's also preoccupied with returning data that's related only to one specific component that uses it.